docs/modules/player-lifecycle/heartbeating.md

```diff
@@ -0,0 +1,43 @@
+<%(TOC)%>
+# Heartbeating
+
+Heartbeating is a technique used to regularly checking-in on the health of connected client-workers and ensuring that unresponsive or disconnected clients are periodically removed from the game world.
+
+## Why it's useful
+
+Client-workers typically create a new Player entity at some point after connecting to SpatialOS. Without the heartbeats, this Player entity would simply exist in the world forever as client-workers would create a different, new Player entity on each new connection.
+
+By introducing heartbeats, we can ensure that a Player entity corresponding to a disconnected or unresponsive client-worker is deleted from the world. This assures us that, in a stable deployment, the number of Player entities in a world is equal to the number of client-workers connected.
```

Comment:
You can have multiple player entities per client. I think the invariant is better expressed as: each Player entity in the world has a client-worker that "owns" it connected or has been connected in the last X seconds. 

Ofc this then brings into question ownership...